src/main/kotlin/no/nav/familie/ks/sak/common/util/Utils.kt

```diff
@@ -8,3 +8,5 @@ inline fun <reified T : Enum<T>> konverterEnumsTilString(liste: List<T>) = liste
 
 inline fun <reified T : Enum<T>> konverterStringTilEnums(string: String?): List<T> =
     if (string.isNullOrBlank()) emptyList() else string.split(";").map { enumValueOf(it) }
+
+fun slåSammen(values: List<String>): String = Regex("(.*),").replace(values.joinToString(", "), "$1 og")
```

Comment:
```suggestion
fun slåSammen(listeMedString: List<String>): String = Regex("(.*),").replace(values.joinToString(", "), "$1 og")
```
Eller noe sånt istedenfor values 😛

Reply:
Godt poeng!